### PR TITLE
Bump Easymde to v2.9.1 314.0

### PR DESCRIPTION
**Pull request by StefKors**
This PR updates Easymde to version `2.9.1 314.0`. This updated version fixes a z-index issue on the `.CodeMirror` class. 

More information: https://github.com/Ionaru/easy-markdown-editor/pull/175

Let me know if there is anything I can do to improve this PR and get a new version published.

### CODE REVIEW

**Comment by StefKors** [comment]
I've added a commit which updates the outdated packages. With was needed due to multiple fsevents errors. Currently `yarn prebuild:lib` is still failing with the following error

```shell
❯ yarn prebuild:lib
yarn run v1.19.2
$ tsc
node_modules/easymde/types/easymde.d.ts:23:23 - error TS2688: Cannot find type definition file for 'marked'.

23 /// <reference types="marked"/>
                         ~~~~~~

node_modules/easymde/types/easymde.d.ts:91:25 - error TS2503: Cannot find namespace 'marked'.

91         markedOptions?: marked.MarkedOptions;
                           ~~~~~~


Found 2 errors.

error Command failed with exit code 2.
```

I'm investigating further what needs to be updated/fixed to solve this. From the looks of easymde might need another fix.

File: yarn.lock [hunk 1]
```diff
@@ -3261,54 +3584,62 @@ domutils@1.5.1:
...
+easymde@^2.9.1-314.0:
+  version "2.9.1-314.0"
+  resolved "https://registry.yarnpkg.com/easymde/-/easymde-2.9.1-314.0.tgz#8ab8681ff23941ad68e80e3a44dddc8bfb5c4d5a"
+  integrity sha512-pyCleNMGozyMa0kzUmYTVVDraygtgbomnnuA47t9FaUuZVAtXB5hUrtcMWR4hx429rfvjvA+GXFRpab2vg6vig==
   dependencies:
-    codemirror "^5.48.0"
+    codemirror "^5.52.2"
     codemirror-spell-checker "1.1.2"
-    marked "^0.7.0"
+    marked "^0.8.0"
```
Comment (A-312) [inline yarn.lock]: You have right installation work fine with `marked "^0.8.2"`.

Comment (StefKors) [reply yarn.lock]: @A-312 it seems that even with the update to marked 0.8.2 the build is still failing. I'll spend some time next week learning more about typescript to get it working. 

Could you explain how you got the installation to work without issues?

Comment (A-312) [reply yarn.lock]: @StefKors I edited this line of my yarn.lock

Comment (A-312) [reply yarn.lock]: @StefKors Sorry I can't reproduce, this fail (i retried)
```
git clone https://github.com/RIP21/react-simplemde-editor/
cd react-simplemde-editor\
git pr 119
yarn install
# failed
```

Comment (StefKors) [reply yarn.lock]: @A-312 Thanks for confirming, I'll continue digging into why this happens

**Comment by RIP21** [comment]
Hey. LGTM.
Why CI checks are failing tho?

**Comment by StefKors** [comment]
@RIP21 Seems like typescript resolutions are failing. Removing the `traceResolution` flag I added to see if that fixes the resolution.

**Comment by StefKors** [comment]
CI seems to fail [here](https://app.netlify.com/sites/react-simplemde-edtior/deploys/5ea6daa1e7f8090006d57ed7)
```
https://app.netlify.com/sites/react-simplemde-edtior/deploys/5ea6daa1e7f8090006d57ed7
```

Seems like clearing the CI cache might fix it:https://github.com/facebook/create-react-app/issues/8810#issuecomment-616843299

**Comment by StefKors** [comment]
@RIP21 Could you review this PR?

**Comment by RIP21** [comment]
@StefKors done. I'll publish later today probably. I'll let you know.
Thanks for the contribution!

**Comment by StefKors** [comment]
@RIP21 Did you have time to publish a new release?

**Comment by RIP21** [comment]
@StefKors sorry for the delay. I'm the worst :D 

Published 4.1.1.

Test, if any issues with this publication let me know :)

**Pull request merged**